src/r2b2/tests/util.py

```diff
@@ -0,0 +1,83 @@
+import json
+from random import randint
+
+from r2b2.contest import Contest
+from r2b2.contest import ContestType
+from r2b2.election import Election
+
+
+def generate_contest(max_size):
```

Comment:
The hypothesis library is great for this sort of thing, and even tries to dig out corner cases in your code.
https://hypothesis.readthedocs.io/en/latest/

Perhaps we should some day move to using it, and put those tests in the r2b2/tests directory, and just hardcode a single testcase here.